### PR TITLE
Fixed pvwatts_ac model bug using module pdc0 instead of inverted pdc0

### PR DESCRIPTION
**Pull request by JPalakapillyKWH**
- [x] Closes issue #734 
 - [x] I am familiar with the [contributing guidelines](http://pvlib-python.readthedocs.io/en/latest/contributing.html).
 - [x] Fully tested. Added and/or modified tests to ensure correct behavior for all reasonable inputs. Tests (usually) must pass on the TravisCI and Appveyor testing services.
 - [x] Adds description and name entries in the appropriate `docs/sphinx/source/whatsnew` file for all changes.
 - [x] Code quality and style is sufficient. Passes LGTM and SticklerCI checks.
 - [x] New code is fully documented. Includes sphinx/numpydoc compliant docstrings and comments in the code where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.

See Issue #734

### CODE REVIEW

**Comment by JPalakapillyKWH** [comment]
@cwhanse Not sure how to add you as a reviewer, or if I need to. I think this should fix the bug though.

**Comment by cwhanse** [comment]
> @cwhanse Not sure how to add you as a reviewer, or if I need to. I think this should fix the bug though.

I'll review.  I think the reviewer assignment feature requires Maintainer privilege, or something.

File: pvlib/pvsystem.py [hunk 1]
```diff
@@ -2849,7 +2853,7 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
     temp_cell: numeric
         Cell temperature in degrees C.
     pdc0: numeric
-        Nameplate DC rating.
+        Nameplate DC rating of the module.
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: Power at standard test condition (STC)

File: pvlib/pvsystem.py [hunk 2]
```diff
@@ -2839,6 +2839,10 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
 
         P_{dc} = \frac{G_{poa eff}}{1000} P_{dc0} ( 1 + \gamma_{pdc} (T_{cell} - T_{ref}))
 
+    Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_ac`. pdc0
+    in this function refers to the nameplate DC rating of the modules. pdc0 in
+    :py:func:`pvwatts_ac`. refers to the nameplate DC rating of the inverter.
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: Editorial:  :py:func:`pvwatts_ac` refers to the DC power input limit of the inverter.

File: pvlib/pvsystem.py [hunk 3]
```diff
@@ -2942,12 +2946,16 @@ def pvwatts_ac(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
 
     where :math:`\zeta=P_{dc}/P_{dc0}` and :math:`P_{dc0}=P_{ac0}/\eta_{nom}`.
 
+    Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_dc`. pdc0
+    in this function refers to the nameplate DC rating of the inverter. pdc0
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: Editorial: :py:func:pvwatts_ac refers to the DC power input limit of the inverter.

File: pvlib/pvsystem.py [hunk 4]
```diff
@@ -2839,6 +2839,10 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
 
         P_{dc} = \frac{G_{poa eff}}{1000} P_{dc0} ( 1 + \gamma_{pdc} (T_{cell} - T_{ref}))
 
+    Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_ac`. pdc0
+    in this function refers to the nameplate DC rating of the modules. pdc0 in
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: `in this function refers to the DC power of the modules at reference conditions.

File: pvlib/pvsystem.py [hunk 5]
```diff
@@ -2942,12 +2946,16 @@ def pvwatts_ac(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
 
     where :math:`\zeta=P_{dc}/P_{dc0}` and :math:`P_{dc0}=P_{ac0}/\eta_{nom}`.
 
+    Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_dc`. pdc0
+    in this function refers to the nameplate DC rating of the inverter. pdc0
+    in :py:func:`pvwatts_dc`. refers to the nameplate DC rating of the modules.
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: Edit to be consistent with the note in pvwatts_dc

File: pvlib/pvsystem.py [hunk 6]
```diff
@@ -2942,12 +2946,16 @@ def pvwatts_ac(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
...
+
     Parameters
     ----------
     pdc: numeric
         DC power.
     pdc0: numeric
-        Nameplate DC rating.
+        Nameplate DC rating of the inverter.
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: DC input limit of the inverter

File: pvlib/pvsystem.py [hunk 7]
```diff
@@ -2947,15 +2948,16 @@ def pvwatts_ac(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
     where :math:`\zeta=P_{dc}/P_{dc0}` and :math:`P_{dc0}=P_{ac0}/\eta_{nom}`.
 
     Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_dc`. pdc0
-    in this function refers to the nameplate DC rating of the inverter. pdc0
-    in :py:func:`pvwatts_dc`. refers to the nameplate DC rating of the modules.
+    in this function refers to he DC power input limit of the inverter.
```
Comment (cwhanse) [inline pvlib/pvsystem.py]: spelling "to the DC power" then good to merge, I think.

**Comment by JPalakapillyKWH** [comment]
stickler has been queued forever. Not sure what's going on here. Manual rebuild?

**Comment by cwhanse** [comment]
Must have gotten stuck

**Comment by cwhanse** [comment]
One last request: please add a description to the Bug Fixes section of whatsnew/v0.6.4.rst, and add your name to the contributors list in that file.

**Comment by cwhanse** [comment]
thanks @JPalakapillyKWH !

**Pull request merged**